test/parallel/test-assert-async.js

```diff
@@ -93,9 +93,11 @@ promises.push(assert.rejects(
     assert(err instanceof assert.AssertionError,
            `${err.name} is not instance of AssertionError`);
     assert.strictEqual(err.code, 'ERR_ASSERTION');
+    assert.strictEqual(err.operator, 'doesNotReject');
```

Comment:
Duplicated with the second line below ?

https://github.com/nodejs/node/blob/eb2d4161f56efa7d82a6364734c674757f1f3ea4/test/parallel/test-assert-async.js#L98

Reply:
Good catch :+1: